torch/_inductor/fx_passes/pad_mm.py

```diff
@@ -391,6 +391,8 @@ def pad_bmm(mat1, mat2, m_padded_length, k_padded_length, n_padded_length):
 def _pad_mm_init():
     from .joint_graph import patterns
 
+    counters_ref = counters["inductor"].copy()
```

Comment:
Perhaps this should go in the lazy_init decorator (and we should use that decorator for this function).

Reply:
cc: @eellison

Reply:
sgtm